recipes/qwt/all/conanfile.py

```diff
@@ -63,7 +64,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("qt/5.15.11", transitive_headers=True, transitive_libs=True)
+        self.requires("qt/[>=5.15.11 <7]", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("qt/6.6.2", transitive_headers=True, transitive_libs=True)
```
Version ranges are only allowed for a small subset of packages currently and Qt is not one of them.